src/modules/Providers/views/details/tabs/VirtualMachines/OpenShiftVirtualMachinesList.tsx

```diff
@@ -53,7 +53,7 @@ export const openShiftVmFieldsMetadataFactory: ResourceFieldFactory = (t) => [
     filter: {
       placeholderLabel: t('Filter by features'),
       type: 'features',
-      values: EnumToTuple(toVmFeatureEnum(t)),
+      values: enumToTuple(toVmFeatureEnum(t)),
```

Comment:
no need to pass t it can import directly

Reply:
This is an old pre-refactored file, I just fixed the utility method name .